src/fcast.c

```diff
@@ -3,6 +3,13 @@
 // #include <signal.h> // the debugging machinery + breakpoint aidee
 // raise(SIGINT);
 
+inline bool any_NA_int(const int n, const int *idx) {
+  for (int i=0; i<n; ++i) {
+    if (idx[i] == NA_INTEGER) return true;
```

Comment:
Slightly confused why we only need to look for `NA_INTEGER`, won't we ever need to look for non-`int` objects in the fill? Maybe I'm not fully understanding the approach/how it fits in to normal execution of `dcast()`.

Reply:
fcast has argument SEXP idxArg which is INTEGER describing which data to reshape (NA if not used)